vendor/github.com/pmezard/go-difflib/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2013, Patrick Mezard
```

Comment:
not sure about this license but I see it's already included in the k8s.io vendor tree